### DISCUSSION TITLE
Where I can find REST API docs?

### DISCUSSION THREAD

**Discussion by Dzordzu** [globaleaks/globaleaks-whistleblowing-software#3620] [Q&A]
REST API section is missing in the documentation. Is there any swagger, or any other tool/place that provides information about this functionality? I'm willing to write an terraform (and/or) ansible module, but I couldn't find any kind of help in this area

**Comment by evilaliv3** [comment]
Hello @Dzordzu , thank  you for your question.

At the moment the API is not documented because it is actually considered an internal API between the client and the backend and we did not found an easy way to document it with a swagger.

I would be happy to discuss with you about this topic and see if and how we could achieve full documentation of the API that could foster next development.

If interested please reach me and our community on our slack: https://slack.globaleaks.org
